From efea9d64fbda52e4a6db210196c925eb3752c1ce Mon Sep 17 00:00:00 2001
From: Kai Xing <kxing@codeaurora.org>
Date: Mon, 17 May 2021 16:24:03 +0530
Subject: [PATCH] gst/gstpad.c: avoid more warning logs in gstpad

Update g_waring to GST_WARNING for control log by GST_DEBUG

Signed-off-by: Kai Xing <kxing@codeaurora.org>
---
 gst/gstpad.c | 18 +++++-------------
 1 file changed, 5 insertions(+), 13 deletions(-)

diff --git a/gst/gstpad.c b/gst/gstpad.c
index dcd52f2..bbafa93 100644
--- a/gst/gstpad.c
+++ b/gst/gstpad.c
@@ -4288,14 +4288,10 @@ gst_pad_chain_data_unchecked (GstPad * pad, GstPadProbeType type, void *data)
 #ifdef GST_ENABLE_EXTRA_CHECKS
   if (G_UNLIKELY (pad->priv->last_cookie != pad->priv->events_cookie)) {
     if (!find_event_by_type (pad, GST_EVENT_STREAM_START, 0)) {
-      g_warning (G_STRLOC
-          ":%s:<%s:%s> Got data flow before stream-start event",
-          G_STRFUNC, GST_DEBUG_PAD_NAME (pad));
+      GST_WARNING("Got data flow before stream-start event");
     }
     if (!find_event_by_type (pad, GST_EVENT_SEGMENT, 0)) {
-      g_warning (G_STRLOC
-          ":%s:<%s:%s> Got data flow before segment event",
-          G_STRFUNC, GST_DEBUG_PAD_NAME (pad));
+      GST_WARNING("Got data flow before segment event");
     }
     pad->priv->last_cookie = pad->priv->events_cookie;
   }
@@ -4541,14 +4537,10 @@ gst_pad_push_data (GstPad * pad, GstPadProbeType type, void *data)
 #ifdef GST_ENABLE_EXTRA_CHECKS
   if (G_UNLIKELY (pad->priv->last_cookie != pad->priv->events_cookie)) {
     if (!find_event_by_type (pad, GST_EVENT_STREAM_START, 0)) {
-      g_warning (G_STRLOC
-          ":%s:<%s:%s> Got data flow before stream-start event",
-          G_STRFUNC, GST_DEBUG_PAD_NAME (pad));
+      GST_WARNING("Got data flow before stream-start event");
     }
     if (!find_event_by_type (pad, GST_EVENT_SEGMENT, 0)) {
-      g_warning (G_STRLOC
-          ":%s:<%s:%s> Got data flow before segment event",
-          G_STRFUNC, GST_DEBUG_PAD_NAME (pad));
+      GST_WARNING("Got data flow before segment event");
     }
     pad->priv->last_cookie = pad->priv->events_cookie;
   }
-- 
2.7.4

